resources/views/editproduct.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="row">
+            <div class="col-md-8 col-md-offset-2">
+                <div class="panel panel-default">
+                    <div class="panel-heading">Edit Product</div>
+
+                    <div class="panel-body">
+                        <form class="form-horizontal" method="POST" action="/products/edit/{{ $id }}">
+                            {{--<input type="hidden" name="_method" value="PATCH">--}}
```

Comment:
Why did you comment the patch method? You can always use `Route::patch` instead of `Route::post` in your routes file